plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestShowStats.java

```diff
@@ -36,6 +37,7 @@ protected QueryRunner createQueryRunner()
                 // create nation so tpch schema got created
                 .setInitialTables(ImmutableList.of(NATION))
                 .setNodeCount(1)
+                .setExtraProperties(ImmutableMap.of("optimizer.default-filter-factor-enabled", "false"))
```

Comment:
Could we update tests instead?

Reply:
I've updated tests to disable default filter factor for only the one query where it is relevant now